libs/components/config/src/lib/app-config-host.ts

```diff
@@ -15,14 +15,14 @@ const DEFAULTS: SkyuxConfigHost = {
  */
 @Injectable()
 export class SkyAppConfigHost {
-  public get host(): SkyuxConfigHost {
-    return this._host;
+  public get host(): SkyuxConfigHost | undefined {
```

Comment:
While this is functionally equivalent to what was there before, it will present a breaking change for consumers using strict mode. I think it might be better to just initialize this property with a default value so it's never undefined. The likelihood that it will ever be accessed before `init()` is called is low, and if it is, only code explicitly checking for `undefined` will cause a problem, and that narrows the likelihood down even more.

Reply:
Talked and did a non-null here.